config/schema/field_definitions.json

```diff
@@ -84,6 +84,11 @@
     "type": "float"
   },
 
+  "popularity_b": {
+    "description": "Popularity rank used to bias search results. Used at query time to compute a popularity score. A higher number indicates more pageviews. Updated nightly.",
+    "type": "float"
```

Comment:
This is going to be an integer, is there an integer type we can use?

Reply:
Changed it to integer